### DISCUSSION TITLE
错误

### DISCUSSION THREAD

**Discussion by bocchi-manGo** [moesnow/March7thAssistant#287] [Q&A]
<img width="300" alt="屏幕截图 2024-07-02 215538" src="https://github.com/moesnow/March7thAssistant/assets/148430895/4659d785-b951-4e63-9db2-6622fe11cd55">
<img width="300" alt="屏幕截图 2024-07-02 215538" src="https://github.com/moesnow/March7thAssistant/assets/148430895/4659d785-b951-4e63-9db2-6622fe11cd55">

**Comment by w593156367** [comment]
![Uploading 微信截图_20250225144900.png…]()

**Comment by w593156367** [comment]
![微信截图_20250225144900](https://github.com/user-attachments/assets/75ee65f8-d318-4e4e-aa75-2b0db302828e)

**Reply by awsl1110** [reply]
好像是安全中心报毒把东西删了，去安全中心操作选择保留就行
